File: src/main/java/me/grian/griansbetamod/events/init/InitListener.java
package me.grian.griansbetamod.events.init;

import net.mine_diver.unsafeevents.listener.EventListener;
import net.modificationstation.stationapi.api.event.mod.InitEvent;
import net.modificationstation.stationapi.api.mod.entrypoint.Entrypoint;
import net.modificationstation.stationapi.api.util.Null;
import net.modificationstation.stationapi.api.util.Namespace;
import org.apache.logging.log4j.Logger;

public class InitListener {

    @Entrypoint.Namespace
    public static Namespace NAMESPACE = Null.get();

    @Entrypoint.Logger
    public static Logger LOGGER = Null.get();

    @EventListener
    public static void serverInit(InitEvent event) {
        LOGGER.info(NAMESPACE.toString());
    }
}

File: src/main/kotlin/me/grian/griansbetamod/events/init/InitListener.kt
package me.grian.griansbetamod.events.init

import net.mine_diver.unsafeevents.listener.EventListener
import net.modificationstation.stationapi.api.event.mod.InitEvent
import net.modificationstation.stationapi.api.util.Namespace
import org.apache.logging.log4j.Logger

object InitListener {
    val NAMESPACE: Namespace = Namespace.of("griansbetamod")

    val LOGGER: Logger = NAMESPACE.logger

    @EventListener
    fun serverInit(event: InitEvent?) {
        LOGGER.info(NAMESPACE.toString())
    }
}